backend/app/services/chat/trigger/core.py

```
@@ -1837,3 +1858,115 @@ def _get_bot_mcp_servers_for_http(bot_name: str, bot_namespace: str) -> Dict[str
         return {}
     finally:
         db.close()
+
+
+async def _trigger_device_execution(
+    task: TaskResource,
+    assistant_subtask: Subtask,
+    team: Kind,
+    user: User,
+    payload: Any,
+    task_room: str,
```

Ruff (0.14.14)

Unused function argument: `task_room`

(ARG001)